### PR TITLE
Make Chip component configurable

### PR DESCRIPTION
**Pull request by annafatani**
Create a chip component consisting of:
1. Basic chip with no action
2. Dismissable chip

### CODE REVIEW

File: src/lib/Chip.svelte [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+	import type { SvelteHTMLElements } from 'svelte/elements';
+	import { twMerge } from 'tailwind-merge';
+
+	type $$Props = SvelteHTMLElements['section'];
+
+	let className: $$Props['class'] = undefined;
+	export { className as class };
+</script>
+
+<section
```
Comment (jamesbirtles) [inline src/lib/Chip.svelte:11]: Not sure the `section` is really correct for a chip, probably just a div(?)

File: src/routes/(docs)/Chip/Chip.test.ts
Comment (jamesbirtles) [inline src/routes/(docs)/Chip/Chip.test.ts:1]: should probably keep the tests of components alongside the components themselves in the file tree

File: src/routes/(docs)/Chip/ExampleChip.svelte [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+	import { Chip, Button, Typography } from '$lib';
+</script>
+
+<Chip>
+	<Typography>2 minutes ago</Typography>
```
Comment (jamesbirtles) [inline src/routes/(docs)/Chip/ExampleChip.svelte:6]: The typography component with no variant doesn't do anything so probably don't need to use it here

File: src/routes/(docs)/Chip/ExampleChip.svelte [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+</Chip>
+
+<Chip class="rounded-full border border-gray-300 bg-gray-200 text-gray-800 dark:bg-gray-100">
+	<div class="flex items-center">
+		<Typography class="text-gray-700">Open Now</Typography>
```
Comment (jamesbirtles) [inline src/routes/(docs)/Chip/ExampleChip.svelte:11]: same comment here

**Review by jamesbirtles — CHANGES_REQUESTED** [review]

File: src/lib/chip/Chip.svelte [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+	import type { SvelteHTMLElements } from 'svelte/elements';
+	import { twMerge } from 'tailwind-merge';
+
+	type $$Props = SvelteHTMLElements['section'];
```
Comment (jamesbirtles) [inline src/lib/chip/Chip.svelte:5]: missed changing this to div 🙂

**Review by jamesbirtles — CHANGES_REQUESTED** [review]
Looks good just the one minor thing

**Review by jamesbirtles — APPROVED** [review]

**Review by danny-people-plus — APPROVED** [review]

**Pull request merged**